src/web/components/SharingPermission/ParticipantSearchBar.scss

```diff
@@ -19,6 +19,11 @@
 .clicked .search-bar-input-container {
   background: var(--theme-search-bar-background);
   border-radius: 10px 10px 0 0;
+  border: 2px solid var(--theme-search-border);
```

Comment:
The design looks like it needs the border around the dropdown as well, and Kristen mentioned adding it to both. It's probably good to fix them both together :)